src/vos/ilog.h

```diff
@@ -33,9 +33,14 @@
 
 struct ilog_id {
 	/** DTX of entry */
-	uint32_t	 id_tx_id;
-	/** Padding */
-	uint32_t	 id_pad1;
+	union {
+		uint64_t	id_value;
+		struct {
+			uint32_t	 id_tx_id;
+			uint16_t	 id_punch_minor_eph;
+			uint16_t	 id_update_minor_eph;
+		};
+	};
```

Comment:
we defined it for twice...

Reply:
I'm not sure if I understand the comment but I did make an intentional decision to not put the prec struct here to avoid too many changes in one patch.  Ideally we would use the same struct